### PR TITLE
DM-42744: Fix random seeds for `ap_association` unit tests

### PR DESCRIPTION
**Pull request by BrunoSanchez**
Set up random generator seed in two places, main location is the `utils_tests.py` that is reused broadly.

## Checklist

- [x] ran Jenkins
- [x] ran local unit tests

### CODE REVIEW

File: tests/utils_tests.py [hunk 1]
```diff
@@ -32,6 +32,9 @@
 import lsst.daf.base as dafBase
 import lsst.geom
 
+# Create an instance of random generator with fixed seed.
+rng = np.random.default_rng(1234)
```
Comment (isullivan) [inline tests/utils_tests.py:36]: In my testing, this results in different random catalogs for each test if I randomize the order the tests are run with `pytest --random-order`. I suggest instead defining `rng` in the `setUp` method of each class that needs it, and passing that in to `makeDiaObjects`, `makeDiaSources`, and `makeDiaForcedSources`

**Review by isullivan — CHANGES_REQUESTED** [review]
See the code comment.

**Comment by BrunoSanchez** [comment]
Changes implemented. Ready for new review

File: tests/utils_tests.py [hunk 2]
```diff
@@ -32,8 +32,11 @@
 import lsst.daf.base as dafBase
 import lsst.geom
 
+# # Create an instance of random generator with fixed seed.
+# rng = np.random.default_rng(1234)
```
Comment (isullivan) [inline tests/utils_tests.py:36]: Did you intend for this to be commented out?

**Review by isullivan — APPROVED** [review]
One commented-out line should probably be uncommented, or removed if not needed.

File: tests/utils_tests.py (hunk 2 above)
Comment (BrunoSanchez) [reply tests/utils_tests.py]: No sorry. Thanks for the catch

**Pull request merged**